.eslintrc

```diff
@@ -11,6 +11,7 @@
   "rules": {
     "import/order": "off",
     "import/extensions": "off",
+    "lodash/import-scope": "off",
```

Comment:
Is this to avoid the global import `_` because for some reason they _really_ want you to use it?

Reply:
yea this setting prevent you from doing `import { find } from 'lodash'`, it's a weird default rule